src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -60,6 +60,18 @@ public PersonBuilder withEmail(String email) {
         return this;
     }
 
+    public Name getName() {
+        return name;
+    }
+
+    public Phone getPhone() {
+        return phone;
+    }
+
+    public Email getEmail() {
+        return email;
+    }
+
```

Comment:
Do we need this?

Reply:
No need, will change (together with suggestion below)